app/src/main/java/com/example/tasky/feature_agenda/presentation/Agenda.kt

```diff
@@ -0,0 +1,115 @@
+package com.example.tasky.feature_agenda.presentation
+
+import android.content.Context
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeDrawingPadding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavController
+import com.example.tasky.AuthNavRoute
+import com.example.tasky.R
+import com.example.tasky.common.domain.error.DataError
+import com.example.tasky.common.presentation.CreateErrorAlertDialog
+import com.example.tasky.common.presentation.LoadingSpinner
+
+@Composable
+fun AgendaContent(
+    navController: NavController,
+    agendaViewModel: AgendaViewModel = hiltViewModel()
+) {
+    val uiState by agendaViewModel.uiState.collectAsStateWithLifecycle()
+    val viewState by agendaViewModel.viewState.collectAsStateWithLifecycle()
```

Comment:
What's the difference between view and UI state? Sounds very misleading to me

Reply:
I'm not sure why I put 2 separate. Will merge these